iina/Base.lproj/Localizable.strings

```diff
@@ -167,6 +167,12 @@
 "preference.sub_override_level.strip" = "strip";
 "preference.sub_override_level.yes" = "yes";
 
+"preference.sub_override_level.descriptive_text.no" = "No override: render subtitles as specified by the subtitle scripts.";
```

Comment:
The `No override` prefix is inconsistent with the other descriptions. For this one I would use the exact description from the mpv manual:
`Render subtitles as specified by the subtitle scripts, without overrides.`